frontend/src/views/Search.vue

```diff
@@ -624,6 +665,33 @@ export default class Search extends Vue {
     this.dateInputStart = getDateFromBeginningOfYear()
   }
 
+  pressKey() {
+    window.addEventListener('keydown', e => {
+      if (e.keyCode == 37) this.setPreviousDate()
+      if (e.keyCode == 39) this.setNextDate()
+    })
+  }
+
+  setPreviousDate() {
+    if (this.dateTo > this.beginningOfHistory) {
+      const date = this.dateTo
+      date.setDate(date.getDate() - 1)
+      this.dateTo = date
+      this.dateFrom = date
+      this.defaultVizDate = date
```

Comment:
Tarvitseeko defaultVizDatea muuttaa? Sama alla.

Reply:
Joo, täytyy muuttaa, tai previousdate nappula ei toimi. Jos nyt ymmärrän oikein, niin se ei päivity muualla, vaikka dateTo muuttujaa muuttaa, joten siihen jää väärä arvo, jos sitä ei päivitä tuossa. Date arvo kyllä sinänsä päivittyy, mutta kalenteri ei. Sama alla olevassa

Reply:
Olet oikeassa, `Datepicker`issä on kuuntelija vain tuolle `defaultVizDate`lle. Nappulat näyttäisivät itseasiassa toimivan, vaikka koodista poistaisi rivit

      this.dateTo = date
      this.dateFrom = date

Tuo `defaultVizDate` on tässä tapauksessa aika hämäävä nimi muuttujalle, sillä nythän me ei oikeastaan vaihdeta sitä default-päivää. Muuttujan voisi joko nimetä jotenkin paremmin, tai jos sille ei löydy mitään järkevää nimeä niin ainakin voisi kommentoida koodiin, miksi sitä muutetaan nappuloita painettaessa.